nixos/modules/misc/documentation.nix

```
@@ -51,12 +76,11 @@ let
           (name: value:
             let
               wholeName = "${namePrefix}.${name}";
-              guard = lib.warn "Attempt to evaluate package ${wholeName} in option documentation; this is not supported and will eventually be an error. Use `mkPackageOption{,MD}` or `literalExpression` instead.";
+              guard = warn "Attempt to evaluate package ${wholeName} in option documentation; this is not supported and will eventually be an error. Use `mkPackageOption{,MD}` or `literalExpression` instead.";
             in if isAttrs value then
               scrubDerivations wholeName value
               // optionalAttrs (isDerivation value) {
                 outPath = guard "\${${wholeName}}";
```

Can't make a suggestion on the deleted line, so on a new comment. Let's fix the bug:

```suggestion
                outPath = guard "\${${wholeName}}";
                drvPath = guard value.drvPath;
```

---

Wonderful, I’ll do that.